app/main/util/date_filters_validator.py

```diff
@@ -18,18 +18,12 @@ def validate_date_filters(args, browse_consignment=False):
         error_field,
     ) = validate_dates(args)
 
-    date_filter_field = args.get("date_filter_field", "")
-
-    if browse_consignment and date_filter_field not in [
-        "date_last_modified",
-        "opening_date",
-    ]:
-        error_field = []
-        date_validation_errors.clear()
-        date_validation_errors["date_filter_field"] = (
-            "Select either ‘Date of record’ or ‘Record opening date’"
+    if browse_consignment:
+        date_validation_errors = _check_consignment_date_filters(
```

Comment:
Instead of needing to pass in date_validation_errors to  `_check_consignment_date_filters` and needing it potentially clear that object, could we separate concerns by calling `_check_consignment_date_filters` before `validate_dates` and only call `validate_dates` if _check_consignment_date_filters didnt result in an error?